src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -400,17 +400,17 @@ protected final boolean passesCallThreshold(final double conf) {
         }
 
         if (doAlleleSpecificCalcs){
-            List<Double> perAlleleQuals = new ArrayList<>();
+            List<Integer> perAlleleQuals = new ArrayList<>();
```

Comment:
Can you comment on the reason for the double -> int change here?

Reply:
It used to be log10 and only used internally, but now it's Phred-scaled because Tim or Nils requested being able to save the raw annotations, in which case the QUALs should be Phred-scaled.